src/quantities.jl

```diff
@@ -307,8 +307,12 @@ round(::Type{T}, u::Units, q::AbstractQuantity, r::RoundingMode=RoundNearest;
     round(Quantity{T, dimension(u), typeof(u)}, q, r; kwargs...)
 
 # workhorse methods
-round(x::AbstractQuantity, r::RoundingMode=RoundNearest; kwargs...) =
+round(x::AbstractQuantity, r::RoundingMode; kwargs...) =
     _rounderr()
+function round(x::AbstractQuantity; sigdigits::Union{Nothing,Integer}=nothing, kwargs...)
+    sigdigits === nothing && _rounderr()
+    round(unit(x), x; sigdigits=sigdigits, kwargs...)
```

Comment:
```suggestion
function round(x::AbstractQuantity, r::RoundingMode=RoundNearest; sigdigits::Union{Nothing,Integer}=nothing, kwargs...)
    sigdigits === nothing && _rounderr()
    round(unit(x), x, r; sigdigits=sigdigits, kwargs...)
```
So that we allow `round(::AbstractQuantity, ::RoundingMode; sigdigits)` as well.